tests/rt_gnu.conf

```diff
@@ -64,8 +66,8 @@ RUN     | fv3_esg_HAFS_v0_hwrf_thompson_debug
 COMPILE | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_2017_coupled                                                                            |                | fv3         |
```

Comment:
This compile will also fail on cheyenne.gnu. Should we just add -chyenne.gnu to cpld_control, cpld_debug and ng-gdas-nemsdatm since they compile and run on hera.gnu

Reply:
Yes, this is what I did in my first round of bug fixes - thanks. Let's see if the bug fixes in FV3GFS_io.F90 resolve the restart issue for the coupled tests.